src/main/java/com/internship/socialnetwork/service/impl/PostServiceImpl.java

```diff
@@ -0,0 +1,62 @@
+package com.internship.socialnetwork.service.impl;
+
+import com.internship.socialnetwork.dto.PostDto;
+import com.internship.socialnetwork.exception.NotFoundException;
+import com.internship.socialnetwork.model.Post;
+import com.internship.socialnetwork.repository.PostRepository;
+import com.internship.socialnetwork.service.PostService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+import static com.internship.socialnetwork.dto.PostDto.convertToDto;
+
+@Service
+@RequiredArgsConstructor
+public class PostServiceImpl implements PostService {
+
+    private final PostRepository postRepository;
+
+    private static final String NOT_FOUND_EXCEPTION_MESSAGE = "Post with id: %1$s not found!";
```

Comment:
Constants should be above injected repos or services, just below class declaration

Reply:
ALso the message should bi renamed to POST_NOT_FOUND or POST_NOT_FOUND_(MSG, ERROR). Whatever you choose but then just follow the same format, not somewhere with _MSG and somewhere without it